src/java.base/share/classes/jdk/internal/icu/text/BidiLine.java

```diff
@@ -47,7 +47,7 @@
      * text in a single paragraph or in a line of a single paragraph
      * which has already been processed according to
      * the Unicode 3.0 Bidi algorithm as defined in
-     * http://www.unicode.org/unicode/reports/tr9/ , version 13,
+     * http://www.unicode.org/reports/tr9/ , version 13,
```

Comment:
Previous references provided links e.g. "<a href="http://www.unicode.org/unicode/reports/tr9/">Unicode Standard Annex #9</a>" instead of URL.